src/mutations/Metadata.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * This file is part of ciboard
+
+ * Copyright (c) 2021, 2022 Andrei Stepanov <astepano@redhat.com>
+ * 
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ * 
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+import { gql } from '@apollo/client';
+
+export const MetadataUpdate = gql`
+    mutation MetadataUpdate(
+        $_id: ID
```

Comment:
You wrote that you are not sure.
Argument can be optional.
@mgrabovsky I think we have an issue in merge-review.
I am not happy how it goes.

Reply:
>  `This makes it quite hard to write maintainable code on the front-end for other people as well.` 
Prove this.

> `This should be three different mutations.`
Prove this.

> `Why wasn't this design decision discussed?`
I do not want to contribute to this project any more.
You provoke me each time.

Reply:
@Andrei-Stepanov , It is important to answer Matej's question so he can learn from you how the code works:

> With this information I'm not sure how the whole mutation works. How do I modify a record then, if there are only two options – one is to create and one is to delete? Why have a single method for creation, modification and deletion of a record? This should be three different mutations.

This could also be added as a comment to this part of a code, so it is clear what happens when id is null and not-null. I'm also not sure how metadata can be updated? Do we always remove it and create another one?

Reply:
We agreed to check how it will work, and take improvements accordingly if there is a way for improvement.